api/candlepin-api-spec.yaml

```diff
@@ -8909,6 +8911,16 @@ components:
               items:
                 $ref: "#/components/schemas/EnvironmentContentDTO"
 
+    ExportResultDTO:
+      description: Represents the result of an export job
+      properties:
+        exportedConsumer:
+          type: string
+        exportId:
+          type: string
+        href:
+          type: string
+
```

Comment:
I added the ExportResultDTO so that we can deserialize the ExportResult that returned in the resultData field on the AsyncJobStatusDTO object when we use the `exportDataAsync` api method.